.github/workflows/docker.yaml

```diff
@@ -9,47 +11,40 @@ jobs:
   docker:
     strategy:
       matrix:
-        os: [ ubuntu-20.04 ]
-        repository: [ inputoutput/hydra ]
         target: [ hydra-node, hydra-tui ]
 
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
```

Comment:
Suggestion: I would rather pin down the OS version, even though 20.04 (or 21.04) is already quite wide a target.  I would not want bad and puzzling surprises from breaking changes in underlying OS.

Reply:
This build should work on any OS where a `docker` deamon is available.. that's why I deliberately loosened the requirement